pallet/account-migration/src/lib.rs

```diff
@@ -54,11 +54,13 @@ use darwinia_staking::Ledger;
 use dc_primitives::{AccountId as AccountId20, Balance, BlockNumber, Index};
 // substrate
 use frame_support::{
-	log,
+	log, migration,
 	pallet_prelude::*,
+	Blake2_128Concat, StorageHasher,
```

Comment:
IIRC, `Blake2_128Concat` was in `pallet_prelude`.

Reply:
You are right. fixed.